src/main/java/org/skriptlang/skript/bukkit/spark/expressions/ExprCPUUsage.java

```diff
@@ -0,0 +1,98 @@
+package org.skriptlang.skript.bukkit.spark.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.SyntaxStringBuilder;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import me.lucko.spark.api.statistic.StatisticWindow.CpuUsage;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.spark.SparkUtils;
+
+@Name("CPU Usage")
+@RequiredPlugins("Paper 1.21+ or Spark")
+@Description({
+	"Returns the CPU usage readings for either the process (the server) or the system " +
+	"(the machine the server is running on). The expression defaults to returning the process readings.",
+	"The readings are the usage percentage of the system's or process's CPU resources " +
+	"(1.0 means 100%). Readings can be made of the last 10 seconds, last minute, and last 15 minutes.",
+	"This expression can only be used if the server has the Spark plugin installed or is running Paper 1.21+, " +
+	"which includes Spark."
+})
+@Examples({
+	"broadcast cpu usage",
+	"if system cpu usage over the last minute > 90%:",
+		"\tsend \"The server is low on CPU power!\" to console"
+})
+@Since("INSERT VERSION")
+public class ExprCPUUsage extends SimpleExpression<Double> {
```

Comment:
Is there a standard naming convention? I see some `MSPT` and some `Mspt`.

Reply:
?? Did you do the wrong file or something?

Reply:
No, CPU and Cpu is also susceptible to the same convention.

Reply:
Oh okay, but it would of been nice if you made 2 or actually pointed out that CPU also had the same issue, as I was genuinely confused where there was mentioning of MSPT/Mspt in ExprCPUUsage.

Reply:
Anything regarding `Cpu` in ExprCPUUsage cannot be changed due to how it is handled in Spark api. So naming convention is out the window. Same with MSPT. I could rename the `MsptStatistic` class to test, but I don't have much hopes.

Reply:
Oh, I only meant your named elements. For example you have some places you use CPU and some places you use Cpu in your own elements (more specific example is that there's two classes, one using MSPT and one using Mspt). The review here was a suggestion to change it to match Skript's convention, of course

Reply:
Ohhhh, that's what you meant, alright I understand now